translation.rst

```diff
@@ -48,24 +48,37 @@ to learn even more. Overall, the process has several steps:
 
 .. _translation-configuration:
 
+Installation
+------------
+
+In applications using :doc:`Symfony Flex </setup/flex>`, run this command to
+install the translator before using it:
```

Comment:
I'm not mentioning this in 4.0: I'm assuming they have Flex installed and have that dir structure. If we don't assume this, then the docs become a mess.